README.md

```diff
@@ -47,6 +79,27 @@ If Proof General complains about a version mismatch, make sure that the shell's
 make clean; make EMACS=/Applications/Emacs.app/Contents/MacOS/Emacs
 ```
 
+## Keeping Proof General up-to-date
+
+### Using MELPA
+
+As explained in the [MELPA documentation](https://melpa.org/#/getting-started), to update all MELPA packages in one go, it is as easy as typing
+<kbd>M-x package-list-packages RET</kbd> then <kbd>r</kbd> (**r**efresh the package list), <kbd>U</kbd> (mark **U**pgradable packages), and <kbd>x</kbd> (e**x**ecute the installs and deletions).
```

Comment:
`, to update all MELPA packages in one go, it is as easy as typing` → `, updating all MELPA packages in one go is as easy as typing`

Reply:
Will these `**` inside of a word work?

Reply:
Hi @cpitclaudel, good question. On second thought it seems that only the **diff** coloring of .md files is buggy, while the GitHub rendering itself (in View mode) is very OK. So I think I could keep it as is.
Also I've done a bit of research ant it seems that the GitHub-flavored Markdown specification does allow intra-word emphasis with `*` or `**` (but obviously not `_` because of typical intra-underscores in variable identifiers). References:
- https://github.github.com/gfm/#emphasis-and-strong-emphasis
- https://github.github.com/gfm/#example-388